### DISCUSSION TITLE
impossible to use V3

### DISCUSSION THREAD

**Discussion by piolet** [serverless/serverless#10590] [Q&A]
Hi all,

I tried the update, but, I don't understand where the problem comes from. Is there a cache somewhere or did I miss a step? As if it was still using the old version of serverless in the package.json :/
![FKI1GzqWUAALfQL](https://user-images.githubusercontent.com/2697871/151655630-190c4a77-68bc-4214-8c1e-542ecffeb144.jpeg)

![FKI1GzoXsAwt07O](https://user-images.githubusercontent.com/2697871/151655540-73e174ec-f8aa-4029-95a7-0dd8f049e05d.jpeg)

Same result if i remove serverless's line in package.json and after `npm clean-cache`
I don't know why, the old standalone version is always used

Do you have an idea ?

**Comment by pgrzesik** [comment]
Hello @piolet - it seems that you still have the global version installed that is `v2` - with `v3` there was a change that the fallback to local `v3` also has to happen from global `v3`. In your case, you will need to upgrade the standalone installation.

**Comment by piolet** [comment]
but I seem to have already installed the global version, but it doesn't change anything obviously

<img width="1627" alt="Capture d’écran 2022-01-29 à 10 48 18" src="https://user-images.githubusercontent.com/2697871/151656313-cb46c755-f889-4e71-8cdf-70df4ab64d8d.png">

**Reply by pgrzesik** [reply]
But it seems like you have global versions installed both via `npm` and as standalone. Please run `which serverless` and see where it points to - I believe it will point to a standalone binary. You can uninstall that standalone binary with `sls uninstall` if you want to use `npm` based installation

**Comment by piolet** [comment]
It's right ! Thanx.

I remove the .serverless folder in my users directory and, when i relaunch deploy command, now, it used v3.

ths for your help

**Reply by pgrzesik** [reply]
Awesome, I'm glad I could help 🙌
